indico/modules/events/timetable/templates/display/compact.html

```diff
@@ -0,0 +1,175 @@
+{% from 'attachments/_display.html' import render_attachments_folders %}
+{% from 'events/display/common/_manage_button.html' import render_manage_button %}
+{% from 'events/display/indico/_common.html' import render_users %}
+
+
+{% set day_list = days|list %}
```

Comment:
why is this needed? is `days` a rrule/generator? in that case i'd rather listify it outside the template (passing generators to templates is a bit weird)